docs/api-references/counts.md

```diff
@@ -0,0 +1,29 @@
+## Counts - API
```

Comment:
`## Counts - API` for 1 API. Please use a header can cover many API. You can use `## Statistic API`